packages/next/test/integration/integration-1.test.js

```diff
@@ -65,7 +65,13 @@ if (parseInt(process.versions.node.split('.')[0], 10) >= 16) {
       }
     }
 
-    expect(lambdas.size).toBe(3);
+    expect(lambdas.size).toBe(2);
```

Comment:
Why are there fewer lambdas?

Reply:
@styfle I think that's because the current rule `app/api/**/*` is only catching `app/api/hello-again/route.js` route, and applying the custom `memory`/`maxDuration` setting to this route only, so it was producing one for `app/api/hello-again/route.js`, one for `app/**/*` (e.g., `page.js`), and another for `pages/api/**/*` === 3

but we're changing the rule to `app/**/*` in this PR so this reduces the extra one and makes it 2